### PR TITLE
DevelopADC and PWM providers added

### PR DESCRIPTION
**Pull request by MahmoudGSaleh**

### CODE REVIEW

File: Microsoft.IoT.Lightning.Providers/Providers/Microsoft.Iot.Lightning.Providers.vcxproj.filters [hunk 1]
```diff
@@ -24,11 +24,10 @@
     <ClInclude Include="Dummy.h">
```
Comment (bfjelds) [inline Microsoft.IoT.Lightning.Providers/Providers/Microsoft.Iot.Lightning.Providers.vcxproj.filters:24]: is dummy still needed after 10586 is released?

File: Microsoft.IoT.Lightning.Providers/AdcConsumer/AdcConsumer.csproj [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{F65085CF-DE77-4A33-8C65-B546B57E2FAB}</ProjectGuid>
+    <OutputType>winmdobj</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>AdcConsumer</RootNamespace>
+    <AssemblyName>AdcConsumer</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.10240.0</TargetPlatformVersion>
```
Comment (bfjelds) [inline Microsoft.IoT.Lightning.Providers/AdcConsumer/AdcConsumer.csproj]: 10586?

File: Microsoft.IoT.Lightning.Providers/AdcConsumer/StartupTask.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            var deferral = taskInstance.GetDeferral();
+
+            if (!LightningProvider.IsLightningEnabled)
+            {
+                // Lightning provider is required for this sample
+                deferral.Complete();
+                return;
+            }
+
+            // Use the PAC9685 PWM provider
```
Comment (bfjelds) [inline Microsoft.IoT.Lightning.Providers/AdcConsumer/StartupTask.cs]: comment refers to PWM rather than adc?

Comment (MahmoudGSaleh) [inline Microsoft.IoT.Lightning.Providers/AdcConsumer/StartupTask.cs]: Fixed.

File: Microsoft.IoT.Lightning.Providers/Providers/Microsoft.Iot.Lightning.Providers.vcxproj.filters (hunk 1 above)
Comment (MahmoudGSaleh) [inline Microsoft.IoT.Lightning.Providers/Providers/Microsoft.Iot.Lightning.Providers.vcxproj.filters:24]: Yes, it needed, as the toolchain fix won't be part of update 1.

File: Microsoft.IoT.Lightning.Providers/AdcConsumer/AdcConsumer.csproj (hunk 2 above)
Comment (MahmoudGSaleh) [inline Microsoft.IoT.Lightning.Providers/AdcConsumer/AdcConsumer.csproj]: Fixed

**Pull request merged**